tests/std/tests/P0645R10_text_formatting_utf8/test.cpp

```diff
@@ -143,8 +139,58 @@ void test_width_estimation() {
         {TYPED_LITERAL(CharT, "\ufe40\u2000\ufe40\x58"), 6},
         {TYPED_LITERAL(CharT, "\ufe40\ufe40\u2000\x58"), 6},
         {TYPED_LITERAL(CharT, "\ufe40\ufe40\ufe40\x58"), 7},
+

```

Comment:
Added a newline here, as ` {TYPED_LITERAL(CharT, "\u0061\u200D\x58"), 2},
        {TYPED_LITERAL(CharT, "\u0061\U0001F1E6\U0001F1E7\u200D\U0001F1E8\u0062\x58"), 5},` were added in a later pr and `// Pick "short" and "long" codepoints` doesn't apply to them.

Reply:
I'll push a commit to cite the new paragraph and drop the confusing "boundaries" explanation. Our test code doesn't have to go to great lengths to explain why it's selecting various test cases; it's nice but not essential.